spec/ruby/library/objectspace/memsize_of_all_spec.rb

```diff
@@ -0,0 +1,21 @@
+require_relative '../../spec_helper'
+require 'objspace'
+
+describe "ObjectSpace.memsize_of_all" do
```

Comment:
The docs of this say:
> This method is only expected to work with C Ruby.

But it seems we already implement it.
Anyway I think adding specs for it is fine, it doesn't seem worse than `ObjectSpace.each_object`.

Reply:
We implement quite a bit more of `objspace` in random ways.